src/qt/sendcoinsdialog.cpp

```diff
@@ -261,10 +261,11 @@ void SendCoinsDialog::on_sendButton_clicked()
 
     QString strFunds = tr("using") + " <b>" + tr("anonymous funds") + "</b>";
     QString strFee = "";
-    recipients[0].inputType = ONLY_DENOMINATED;
 
     if(ui->checkUsePrivateSend->isChecked()) {
-        recipients[0].inputType = ONLY_DENOMINATED;
+        for (SendCoinsRecipient& rcp : recipients) {
```

Comment:
Maybe better to have two variables on top of the first `if` and then a single loop after all the if/else statements that uses the variables? Would probably be easier to read/maintain then the repeated for loops

Reply:
Agree, fixed.